src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/eventsQueryArgsBuilder/filterConverters/dateConverter.js

```diff
@@ -1,11 +1,13 @@
 // @flow
 import { createSelector } from 'reselect';
 import { moment } from 'capture-core-utils/moment';
```

Comment:
I hate to say it again but I want to make sure that if we agree that we could import the moment library as we do with all the other dependencies we also do it here. :-)

Reply:
see comment from earlier.